src/index.ts

```diff
@@ -145,13 +145,18 @@ export function repl() {
         output: process.stdout,
     });
     rl.setPrompt("brs> ");
-
     rl.on("line", line => {
+        if (line.toLowerCase() === "quit" || line.toLowerCase() === "exit") {
+            process.exit();
+        }
         let results = run(line, defaultExecutionOptions, replInterpreter);
         if (results) {
-            results.map(result => console.log(result.toString()));
+            results.map(result => {
+                if (result.toString() !== "invalid") {
```

Comment:
What about when someone runs something like this:
```brightscript
name = invalid
print name
```

Reply:
It works, try it, this results object is not the actual response on the REPL.

Reply:
![image](https://user-images.githubusercontent.com/16841645/63328540-31c7fc80-c2e5-11e9-80e9-ed49cddee6d3.png)

Reply:
Cool, thanks for verifying!